webpack/assets/javascripts/react_app/components/Architectures/Welcome.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl } from '../../common/helpers';
+
+export const WelcomeArchitecture = ({ canCreate }) => {
+  const action = canCreate && {
+    title: __('Create Architecture'),
+    url: foremanUrl('/architectures/new'),
+  };
+  const content = __(`Each entries represent a particular hardware architecture, most commonly <b>x86_64</b> or <b>i386</b>.
```

Comment:
While we're here let's correct the English:
```suggestion
  const content = __(`Each entry represents a particular hardware architecture, most commonly <b>x86_64</b> or <b>i386</b>.
```